openpype/settings/entities/schemas/projects_schema/schemas/schema_anatomy_imageio.json

```diff
@@ -427,6 +427,35 @@
                 }
             ]
         },
+
+        {
```

Comment:
Please move this fro manatomy to fusion settings (regarding issue https://github.com/pypeclub/OpenPype/issues/3865 ).

Reply:
suggestion: `project_anatomy/imageio/fusion` > `project_settings/fusion/imageio`

Reply:
It's now moved to `project_settings/fusion/imageio/ocio/configFilePath`:

![afbeelding](https://user-images.githubusercontent.com/2439881/190652704-f2afdde3-6f89-4370-bd09-ff0be22873fc.png)

I still need to update the Hook..

Does the hook have access to `self.data["project_settings"]` as I quickly refactored in https://github.com/pypeclub/OpenPype/pull/2769/commits/7aa905898e6726e910c05be8db6514e5e656b8c1 ?